src/generators/static/packageFileGenerator.ts

```diff
@@ -253,9 +253,7 @@ function restLevelPackage(packageDetails: PackageDetails) {
   }
 
   if (
-    generateSample &&
-    (model as TestCodeModel)?.testModel?.mockTest?.exampleGroups &&
-    (model as TestCodeModel).testModel!.mockTest!.exampleGroups.length > 0
+    generateSample && hasRLCSamplesGenerated
```

Comment:
if no examples provided why do you we need to generate the sampleConfiguration ?

Reply:
The condition `hasRLCSamplesGenerated` will be truthy if and only if we have examples provided and we could transform to valid sample models. 

So we will not generate sampleConfiguration if there is no examples provided.